tests/test_dfa.py

```diff
@@ -1423,6 +1423,19 @@ def test_len_infinite(self):
         with self.assertRaises(ValueError):
             len(~dfa)
 
+    def test_successor(self):
+        binary = {'0', '1'}
+        language = {'', '0', '00', '000', '010', '100', '110', '010101111111101011010100'}
+        dfa = DFA.from_finite_language(binary, language)
+        expected = sorted(language)
+        actual = list(dfa.successors('', include_input=True))
+        self.assertListEqual(actual, expected)
+
+        self.assertEqual(dfa.successor('000'), '010')
+        self.assertEqual(dfa.successor('0100'), '010101111111101011010100')
+        self.assertIsNone(dfa.successor('110'))
+        self.assertIsNone(dfa.successor('111111110101011'))
+
```

Comment:
Should have a test that uses the DFA for a simple infinite language, like `0*1*`.

Reply:
Done